Standards/scs-xxxx-v1-achieving-certification.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Regulations for achieving SCS-compatible certification
+type: Procedural
+status: Draft
+track: Global
+---
+
+## Introduction
+
+The Sovereign Cloud Stack (SCS) issues certificates with various scopes, among them _SCS-compatible IaaS_ (infrastructure as a service) and _SCS-compatible KaaS_ (Kubernetes as a service).
+
+This document details how a cloud service provider (henceforth also called operator) can attain such a certificate for one of their clouds.
+
+## Motivation
+
+As operator, I want to obtain a certificate with the scope SCS-compatible IaaS or SCS-compatible KaaS.
+
+## Regulations
+
+1. Each certificate issued pertains to a given cloud, a given scope, and a given version of that scope with a fixed expiry date. The certificate is only valid for that cloud and for the time frame that ends on that expiry date.
+
+2. The operator MUST include the official SCS compliance test suite (which does not require admin privileges) in their continuous test infrastructure (e.g., Zuul). The tests MUST be run at least nightly. For public clouds, it is recommended to offer the SCS project access to the infrastructure (ideally, including a Zuul nodepool) so the test suite runs can be triggered continuously by the SCS team. Alternatively, and for non-public clouds, the results (log files) MUST be submitted to SCS by a mechanism of SCS' choice and need to be reproduced again on request by SCS.
+
+   <!-- Initially this will probably be eMail -->
+
+3. If the desired certificate requires manual checks, then the operator MUST offer the SCS project suitable access. Manual checks SHOULD be repeated once every 3 months.
```

Comment:
This is a huge difference between the daily (or nightly) automated tests and those only done every three months. I also think for IaaS at least such tests need to be done whenever a deployment is updated (e.g. config change, security fix, etc..).

Reply:
The constraint here is the capacity of the manual testers. Let's talk about this in a meeting, as you proposed, because it would be good for me to have a better understanding of the deployments you are referring to and how often this would happen. If it's 4 times a year, but not necessarily spaced out evenly, well, then it might work.

Reply:
In SIG meeting on 19.10.23, we decided to run manual tests once per quarter.

Reply:
There is a small difference between "once every three months" (as suggested in the meeting of 2023-09-21) and "once per quarter" (as suggested in the meeting of 2023-10-20), because the former case implies the time distance between the tests is more or less always three months, whereas in the latter case the distance could also be almost zero (if we test near the end of one quarter and then near the beginning of the next quarter) or almost six months (if we test near the beginning of one quarter and near the end of the next). I would like to keep the former case here and let the community speak.

Reply:
IMHO the "SHOULD" case is too weak and I would rather swap this for "Manual checks MUST be repeated at least every 6 month" give that the SCS Project has a 6 month release cycle.

Reply:
Added this question to the next meeting's agenda. https://input.scs.community/2023-scs-sig-standardization

Reply:
Changed the SHOULD into a MUST. Somehow the interval did not get changed, but I think having a "halftime inspection" is not too bad, actually.